src/Datadog.Trace.ClrProfiler.Managed/ExtensionMethods/SpanExtensions.cs

```diff
@@ -13,21 +13,5 @@ internal static string GetHost(this ISpan span)
 
         internal static string GetAbsoluteUrl(this ISpan span)
```

Comment:
I realize you moved `DecorateWebServerSpan` into `Datadog.Trace` for compile-time issues, but I think it would also be good to move these other helpers into the main library

Reply:
They were removed with the purge. This file is no longer.